reportportal_client/service_async.py

```diff
@@ -302,6 +302,12 @@ def update_test_item(self, description=None, tags=None):
     def finish_test_item(self, end_time, status, issue=None):
         logger.debug("finish_test_item queued")
 
+        # check if skipped test should not be marked as "TO INVESTIGATE"
+        to_investigate = issue is None or issue.get("issue_type") == "TI001"
```

Comment:
The same check is done in the syncronous version of this service, but doing some tests looks like that checking only for `issue is None` is not correct since pytest-reportportal always populates the issue with TI (to investigate) when skipped.
To make it work also an issue of type TO_INVESTIGATE needs to be allowed.

String "T1001" is hardcoded which is not great but simpler to understand, if preferred it can be retrieved as
`self.rp_client.get_project_settings()['subTypes']['TO_INVESTIGATE'][0]['locator']`